momepy/tests/test_dimension.py

```diff
@@ -18,67 +18,30 @@ def setup_method(self):
         self.df_buildings["height"] = np.linspace(10.0, 30.0, 144)
 
     def test_Area(self):
-        with pytest.warns(
```

Comment:
Given we test the decorators and can silence these warnings, I am removing explicit checks from individual classes.